pkg/server/server.go

```diff
@@ -115,6 +121,58 @@ func ServeUI(ctx context.Context, logger log.Logger, uiConfig UIConfig, apiServe
 	}
 }
 
+func ServeConnect(ctx context.Context, logger log.Logger, cfg ConnectConfig, deps api.Deps) error {
+	// Create the server handler with both services
+	frontierService := v1beta1connect.NewConnectHandler(deps)
+
+	// Initialize connect handlers
+	frontierPath, frontierHandler := frontierv1beta1connect.NewFrontierServiceHandler(frontierService)
+	adminPath, adminHandler := frontierv1beta1connect.NewAdminServiceHandler(frontierService)
+
+	// Create mux and register handlers
+	mux := http.NewServeMux()
+	mux.Handle(frontierPath, frontierHandler)
+	mux.Handle(adminPath, adminHandler)
+
+	// configure healthcheck
+	// curl --header "Content-Type: application/json" \
+	// --data '{"service":"raystack.frontier.v1beta1.AdminService"}' \
+	// http://localhost:8002/grpc.health.v1.Health/Check
+	checker := connecthealth.NewStaticChecker(
+		"raystack.frontier.v1beta1.FrontierService",
+		"raystack.frontier.v1beta1.AdminService",
+	)
+	mux.Handle(connecthealth.NewHandler(checker))
+
+	// Configure and create the server
+	server := &http.Server{
+		Addr:    fmt.Sprintf(":%d", cfg.Port),
+		Handler: h2c.NewHandler(mux, &http2.Server{}),
+	}
+
+	logger.Info("connect server starting", "port", cfg.Port)
+
+	go func() {
+		<-ctx.Done()
+
+		ctxShutdown, cancel := context.WithTimeout(context.Background(), time.Minute)
```

Comment:
graceful shutdown period is hardcoded for now, do we want to use similar config as mux?